tests/filecheck/mlir-conversion/with-mlir/dialects/llvm/arithmetic.mlir

```diff
@@ -44,4 +44,24 @@ builtin.module {
 
     %ashr = llvm.ashr %arg0, %arg1 : i32
     // CHECK: %{{.*}} = llvm.ashr %{{.*}}, %{{.*}} : i32
+
+    // float arith:
+
+    %fmul = llvm.fmul %f1, %f1 : f32
+    // CHECK: %{{\d+}} = llvm.fmul %2, %2 : f32
```

Comment:
Any reason for not using a pattern capture here (i.e., `{{.*}}` instead of `%2`)?

Reply:
hmm, why are they not `%f1` in the first place?

Reply:
Ah because of `mlir-opt`

Reply:
Likely better to use a numeric capture if you really want to avoid hard coding this
```suggestion
    // CHECK: %{{\d+}} = llvm.fmul %[[#in:]], %[[#in]] : f32
```

Reply:
yes, that's what I meant :)